commit c5c9f6b5501a366d5809faa44e50fad371c14870
Author: Jon Rowe <hello@jonrowe.co.uk>
Date:   Tue Oct 29 14:18:46 2013 +1100

    fixes the build by disabling path traversal protection for this test
    
    Seems that this test is in direct competition with the desired
    behaviour of path traversal protection.

diff --git a/test/routing_test.rb b/test/routing_test.rb
index 410bfda..5584601 100644
--- a/test/routing_test.rb
+++ b/test/routing_test.rb
@@ -90,7 +90,10 @@ class RoutingTest < Test::Unit::TestCase
   end
 
   it "it handles encoded slashes correctly" do
-    mock_app { get("/:a") { |a| a } }
+    mock_app {
+      set :protection, :except => :path_traversal
+      get("/:a") { |a| a }
+    }
     get '/foo%2Fbar'
     assert_equal 200, status
     assert_body "foo/bar"
